src/ontology/mondo-edit.obo

```diff
@@ -70,7 +70,7 @@ property_value: http://xmlns.com/foaf/0.1/homepage http://obofoundry.org/ontolog
 property_value: IAO:0000700 MONDO:0000001
 property_value: IAO:0000700 MONDO:0021125
 property_value: IAO:0000700 MONDO:0042489
-owl-axioms: Prefix(owl:=<http://www.w3.org/2002/07/owl#>)\nPrefix(rdf:=<http://www.w3.org/1999/02/22-rdf-syntax-ns#>)\nPrefix(xml:=<http://www.w3.org/XML/1998/namespace>)\nPrefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)\nPrefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)\n\n\nOntology(\nDeclaration(AnnotationProperty(<http://www.geneontology.org/formats/oboInOwl#consider>))\n\n\nAnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#consider> <http://purl.obolibrary.org/obo/MONDO_0016606> \"\")\n)
+owl-axioms: Prefix(owl:=<http://www.w3.org/2002/07/owl#>)\nPrefix(rdf:=<http://www.w3.org/1999/02/22-rdf-syntax-ns#>)\nPrefix(xml:=<http://www.w3.org/XML/1998/namespace>)\nPrefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)\nPrefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)\n\n\nOntology(\nDeclaration(Class(<http://purl.obolibrary.org/obo/mondo.owl/MONDO_0700005>))\nDeclaration(Class(<http://purl.obolibrary.org/obo/mondo.owl/MONDO_0700006>))\nDeclaration(ObjectProperty(<http://purl.obolibrary.org/obo/RO_0002573>))\nDeclaration(AnnotationProperty(<http://www.geneontology.org/formats/oboInOwl#consider>))\n\n\n\n\nDisjointClasses(ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002573> <http://purl.obolibrary.org/obo/mondo.owl/MONDO_0700005>) ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002573> <http://purl.obolibrary.org/obo/mondo.owl/MONDO_0700006>))\nAnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#consider> <http://purl.obolibrary.org/obo/MONDO_0016606> \"\")\n)
```

Comment:
oh, maybe this is where you tried to add the disjointness axiom? If yes, I don't think this is correct, can you delete it.

The disjointness axiom should be in the Description box:
![image](https://user-images.githubusercontent.com/6722114/123342991-b39e9780-d505-11eb-8a28-09ffdd91934a.png)


you only need to add it to one term, then it will be automatically added to the other.